documentation/tutorials/pomodoro-report.rst

```diff
@@ -0,0 +1,152 @@
+
+Notice that type "Super weird bug" is tedious and error-prone, but don't worry,
+kw provides a feature to list all tags and associate them with an ID. To see
+all tags that you already created, use this command::
+
+  kw p -g
+  1.Super weird bug
+
+In other words, you can use::
+
+  kw p -t 30m -g 1 -d "I know how to reproduce the bug! I'm going to try THIS WORKAROUND to see if it fixes the issue"
+
+You can create as many focus sections as you want at any time and day.
+
+Generate a report
+-----------------
+
+At this point, you are already familiar with the basic functionality around the
+Pomodoro, and you use it daily. However, now someone is asking you for a report
+from your last week, and you worked on so many things that you don't even know
+how to start; don't worry, kw has your back. Now it is time to use the report
+feature. If you want to get this week summary, you can use::
+
+  kw report --week
+
+Or just::
+
+  kw r --week
+
+The report will look like this::
+
+    # Report: YYYY/MM/DDD
+     * Total focus hours: 48:43:00
+     * Total focus section: 23
+
+    ## Super weird bug
+     - Total focus time: 04:20:00
+     - Total repetitions: 22
+
+    Summary:
+     * [15:00:00-15:30:00][30m]: Let's make sure that I know how to reproduce this bug...
+     ...
+     * [17:00:00-17:30:00][30m]: I know how to reproduce the bug! I'm going to try THIS WORKAROUND to see if it fixes the issue
+
+    ## Another Tag
+     - Total focus time: HH:MM:SS
+     - Total repetitions: 4
+
+    Summary:
+     * [08:15:50-08:45:50][30m]: More descriptions
+    ...
+
+As you can see, the report starts with a generic summary, and it is broken down
+per tag. Each tag will become a section, and each description will be put in
+order in the Summary part. This feature allows you to generate reports in the
+following granularity:
+
+* Day: Provide a day using the format YYYY/MM/DD.
+* Week: You need to provide a target day (no matter the weekday), and kw will
+  look for the first and last day associated with that date and generate a report
+  based on that.
+* Month: Provide a month using YYYY/MM and kw will give you that the target
+  month report.
+* Year: Provide the year as YYYY and kw will provide the summary for the entire
+  year.
+
+By default, if you do not provide specific dates, kw report will assume the
+closest date; for example::
+
+    kw report # Shows today's report
+    kw report --week # Shows this week's report
+    kw report --month # Shows this month's report
+    kw report --year # Shows this year's report
+
+Conclusion
+----------
+
+As you can see, Pomodoro and report are very powerful tools to help you track
+your work. If you use this feature regularly, there is a good chance that all
+of this data is valuable for you; in this sense, we recommend you to take a
+look at kw backup feature since it creates a backup of all your Pomodoro data.
```

Comment:
```suggestion
of this data is valuable for you; in this sense, we recommend you take a look
at the kw backup feature since it creates a backup of all your Pomodoro data.
```